.github/dependabot.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+# See https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    commit-message:
+      prefix: "chore:"
+    open-pull-requests-limit: 10
+    rebase-strategy: disabled
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "01:00"
+      timezone: "UTC"
+
+  - package-ecosystem: "docker"
```

Comment:
as the build system is managed by Kres, I don't see any value in doing that for any repo rather than Kres itself => update Kres, re-run it for other repos. We don't want PRs for auto-generated files